doc/internal/man3/ossl_cmp_asn1_octet_string_set1.pod

```diff
@@ -0,0 +1,81 @@
+=pod
+
+=head1 NAME
+
+ossl_cmp_log_parse_metadata,
+ossl_cmp_x509_extensions_dup,
+ossl_cmp_asn1_octet_string_set1,
+ossl_cmp_asn1_octet_string_set1_bytes,
+ossl_cmp_build_cert_chain
+- misc internal utility functions
+
+=head1 SYNOPSIS
+
+ #include "cmp_int.h"
+
+ const char *ossl_cmp_log_parse_metadata(const char *buf,
+                 OSSL_CMP_severity *level, char **func, char **file, int *line);
```

Comment:
style : `OSSL...` should be align with the `const` above.
Same for the code

Reply:
Thx, fixed.